src/components/Hero/HomepageHero.tsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React from 'react';
+import { motion } from 'framer-motion';
+import { Container } from '../Container';
+import { Section } from '../Section';
+import { Heading } from '../Typography';
+
+export const HomepageHero = () => {
+  const lines: string[] = [
+    'Here and now',
+    'is where we',
+    'make the',
+    'world better.',
+    'On purpose.',
+  ];
+
+  // const meshGradient = 'radial-gradient(at 40% 20%, hsla(28,100%,74%,1) 0px, transparent 50%),radial-gradient(at 80% 0%, hsla(189,100%,56%,1) 0px, transparent 50%),radial-gradient(at 0% 50%, hsla(355,100%,93%,1) 0px, transparent 50%),radial-gradient(at 80% 50%, hsla(340,100%,76%,1) 0px, transparent 50%),radial-gradient(at 0% 100%, hsla(22,100%,77%,1) 0px, transparent 50%),radial-gradient(at 80% 100%, hsla(242,100%,70%,1) 0px, transparent 50%),radial-gradient(at 0% 0%, hsla(343,100%,76%,1) 0px, transparent 50%)';
+
+  // const darkMeshGradient = 'radial-gradient(at 86% 46%, hsla(342, 91%, 61%, 1) 0, hsla(342, 91%, 61%, 0) 50%), radial-gradient(at 13% 79%, hsla(325, 85%, 57%, 1) 0, hsla(325, 85%, 57%, 0) 50%), radial-gradient(at 72% 27%, hsla(268, 89%, 65%, 1) 0, hsla(268, 89%, 65%, 0) 50%), radial-gradient(at 79% 3%, hsla(65, 86%, 57%, 1) 0, hsla(65, 86%, 57%, 0) 50%), radial-gradient(at 61% 13%, hsla(201, 85%, 62%, 1) 0, hsla(201, 85%, 62%, 0) 50%), radial-gradient(at 8% 81%, hsla(111, 88%, 63%, 1) 0, hsla(111, 88%, 63%, 0) 50%), radial-gradient(at 46% 70%, hsla(321, 90%, 57%, 1) 0, hsla(321, 90%, 57%, 0) 50%)';
+
+  // const darkMesh3 = 'radial-gradient(at 70% 54%, hsla(317, 85%, 50%, 1) 0, hsla(317, 85%, 50%, 0) 50%), radial-gradient(at 40% 39%, hsla(284, 92%, 70%, 1) 0, hsla(284, 92%, 70%, 0) 50%), radial-gradient(at 9% 94%, hsla(2, 93%, 65%, 1) 0, hsla(2, 93%, 65%, 0) 50%), radial-gradient(at 2% 7%, hsla(264, 87%, 63%, 1) 0, hsla(264, 87%, 63%, 0) 50%)';
+
+  // const darkMesh2 = 'radial-gradient(at 65% 67%, hsla(46, 85%, 65%, 1) 0, hsla(46, 85%, 65%, 0) 50%), radial-gradient(at 100% 33%, hsla(185, 89%, 68%, 1) 0, hsla(185, 89%, 68%, 0) 50%), radial-gradient(at 5% 99%, hsla(279, 89%, 52%, 1) 0, hsla(279, 89%, 52%, 0) 50%), radial-gradient(at 12% 33%, hsla(345, 89%, 62%, 1) 0, hsla(345, 89%, 62%, 0) 50%)';
+
+  // const darkMesh4 = 'radial-gradient(at 50% 14%, hsla(192, 89%, 69%, 1) 0, hsla(192, 89%, 69%, 0) 50%), radial-gradient(at 49% 58%, hsla(343, 94%, 57%, 1) 0, hsla(343, 94%, 57%, 0) 50%), radial-gradient(at 55% 71%, hsla(91, 92%, 67%, 1) 0, hsla(91, 92%, 67%, 0) 50%), radial-gradient(at 1% 13%, hsla(350, 85%, 61%, 1) 0, hsla(350, 85%, 61%, 0) 50%)';
```

Comment:
Commenting these unused gradients out. These are the ones I like, and I'll likely move them into a separate file in the next PR.